modules/reference/nav.adoc

```diff
@@ -77,4 +77,8 @@ include::reference:partial$feature-nav.adoc[]
   ** xref:diff/mp-41-50-diff.adoc[Differences between MicroProfile 5.0 and 4.1]
   ** xref:diff/mp-33-40-diff.adoc[Differences between MicroProfile 4.0 and 3.3]
   ** xref:diff/mp-22-30-diff.adoc[Differences between MicroProfile 3.0 and 2.2]
-  ** xref:diff/mp-21-22-diff.adoc[Differences between MicroProfile 2.2 and 2.1]
\ No newline at end of file
+  ** xref:diff/mp-21-22-diff.adoc[Differences between MicroProfile 2.2 and 2.1]
```

Comment:
this file is the reason there are conflicts in the branch. The nav files have to be edited in specific branches and committed from there- in this case `draft-nav`. See
https://github.com/OpenLiberty/docs#Editing-the-docs-navigation for details